src/atoms/MainRecipe/ColorTextBox.js

```diff
@@ -1,27 +1,16 @@
 import styled from 'styled-components';
 import { Col } from '../../atoms/Col';
+import { getColor } from '../../utils';
 
 const ColorTextBox = styled(Col)`
-	padding: 2rem;
-	width: 100%;
-	height: 100%;
-	display: flex;
-	align-items: center;
+	padding: 3rem;
 	font-size:   ${props => props.theme.variables.headingRegularSize};
 	line-height: ${props => props.theme.variables.headingRegularLineHeight};
-	font-weight: 600;
+	font-weight: 500;
 	letter-spacing: .05rem;
-	text-align: center;
-	justify-content: center;
-	color: ${props => props.theme.colors[props.textColor]};
-	background-color: ${props => props.theme.colors[props.bgColor]};
-
-	@media screen and (max-width: ${props => props.theme.flexboxgrid.breakpoints.md}em) {
-		padding: 4rem 6rem;
-		font-size: 2.2rem;
-		line-height: 1.4;
-		text-align: left;
- 	}
+	text-align: left;
+	color: ${props => getColor(props.textColor)};
```

Comment:
getColor returns a functions that takes props as a parameter - so here you can delete `props =>`

Reply:
This comment is all wrong. Nevermind :)